UE4SS/src/GUI/LiveView.cpp

```
@@ -1817,8 +1840,24 @@ namespace RC::GUI
         Output::send(STR("Searching by name...\n"));
         s_name_search_results.clear();
         s_name_search_results_set.clear();
+
+        uintptr_t address_to_search_by = 0;
+        if (LiveView::s_search_by_address)
+        {
+            try
+            {
+                address_to_search_by = std::stoull(LiveView::s_name_to_search_by, nullptr, 16);
+            }
+            catch (std::invalid_argument) {} // query isn't a valid hex number; can't be an address
+            catch (std::out_of_range) {} // is a hex number, but too big, or negative; can't be address
```

We're silently failing here.
I'd prefer if this was a blocking popup box containing the error message, but will settle for `Output::send<LogLevel::Error>`.

---

> I'm interested in getting this feature merged, so I made this change locally. Is there a preferred way for me to contribute those changes to this PR? e.g. open a new PR, PR into this PR's fork, attach a patch file for a follow-up PR

First step is to push it to your own fork so we can look at it.
We might be able to cherry pick your commit(s).

---

Diff is here: https://github.com/vitrvvivs/RE-UE4SS/compare/search-by-memory-address...ntsik:RE-UE4SS:search-by-memory-address

The base is out-of-date, so the last change doesn't have much context around it in the diff. Off of `main`, I put it in `LiveView::render` underneath the other modals.

---

Thanks. I've cherry picked your commit and rebased the whole thing onto latest main.